src/main/com/fulcrologic/rad/database_adapters/datomic_options.cljc

```diff
@@ -0,0 +1,20 @@
+(ns com.fulcrologic.rad.database-adapters.datomic-options)
+
+
+(def native-id?
+  "If true it will map the given ID attribute (which must be type long) to :db/id."
+  :com.fulcrologic.rad.database-adapters.datomic/native-id?)
+
+(def attribute-schema
+  "If true it will map the given ID attribute (which must be type long) to :db/id."
+  :com.fulcrologic.rad.database-adapters.datomic/attribute-schema)
+
+(def connections
+  "A map, keyed by schema, of the database connection that should be used\nin the context of the current request."
+  :com.fulcrologic.rad.database-adapters.datomic/connections)
```

Comment:
This should not be documented here, since this is for options you'd put on attributes. This is part of the docstring for the middleware.